AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -368,18 +449,15 @@ MAV_RESULT GCS_MAVLINK_Tracker::handle_command_long_packet(const mavlink_command
     switch(packet.command) {
 
     case MAV_CMD_COMPONENT_ARM_DISARM:
-        if (packet.target_component == MAV_COMP_ID_SYSTEM_CONTROL) {
```

Comment:
Because it was stopping disarming tracker from mission planner. I couldn't find any other examples of it in the code base. Maybe you could advise. After removing it seems to work as it should and not disarm when it shouldn't.

Reply:
I cannot advise. <shrug> as just wondering...